tests/integration/hive_metastore/test_tables.py

```diff
@@ -50,3 +53,69 @@ def test_describe_all_tables_in_databases(ws: WorkspaceClient, make_catalog, mak
     assert all_tables[external_table].object_type == "EXTERNAL"
     assert all_tables[view].object_type == "VIEW"
     assert all_tables[view].view_text == "SELECT 2+2 AS four"
+
+
+def test_migrate_view_and_managed_tables(ws, make_catalog, make_schema, make_table):
+    target_catalog = make_catalog()
+    schema_a = make_schema(catalog="hive_metastore")
+    _, target_schema = schema_a.split(".")
+
+    make_schema(catalog=target_catalog, schema_name=target_schema)
+
+    managed_table = make_table(schema=schema_a)
+    view = make_table(schema=schema_a, ctas="SELECT 2+2 AS four", view=True)
+
+    logger.info(f"target catalog={target_catalog}, managed_table={managed_table}, view={view}")
+
+    inventory_schema = make_schema(catalog="hive_metastore")
+    _, inventory_schema = inventory_schema.split(".")
+
+    backend = StatementExecutionBackend(ws, os.environ["TEST_DEFAULT_WAREHOUSE_ID"])
+    workspace_cfg = WorkspaceConfig(
+        default_catalog=target_catalog, groups=GroupsConfig(auto=True), inventory_database=""
+    )
+    tables = TablesCrawler(backend, workspace_cfg)
+    tables.snapshot()
+    tables.migrate_tables()
+
+    target_tables = list(backend.fetch(f"SHOW TABLES IN {target_catalog}.{target_schema}"))
+    assert len(target_tables) == 2
+
+    _, _, view_name = view.split(".")
+    view_property = ""
+    for detail in list(backend.fetch(f"DESCRIBE EXTENDED {view}")):
+        if detail["col_name"] == "Table Properties":
+            view_property = detail["data_type"]
+    assert f"upgraded_to={target_catalog}.{target_schema}.{view_name}" in view_property
```

Comment:
i think that on `target_table` we should check `migrated_from` property, not the others.